src/hiscores.ts

```diff
@@ -64,8 +65,14 @@ export async function getOfficialStats(
   try {
     const response = await httpGet<HiscoresResponse>(url, config);
     return response.data;
-  } catch {
-    throw Error(PLAYER_NOT_FOUND_ERROR);
+  } catch (err) {
+    const error = err as Error | AxiosError;
```

Comment:
You can safely remove this typecast, since the `isAxiosError` will allow the type to be inferred below.